pixl_ehr/src/pixl_ehr/main.py

```diff
@@ -96,6 +97,16 @@ def export_radiology_as_parquet(export_params: ExportRadiologyData) -> None:
     pe.export_radiology(anon_data)
 
 
+@app.post(
+    "/send-parquet-files-via-FTPS",
+    summary="Copy/send all ParquetExports via FTPS",
+)
+def SendViaFTPS(project_name: str, extract_datetime: datetime) -> None:
+    """Send parquet files via FTPS"""
+    pe = ParquetExport(project_name, extract_datetime)
+    upload_parquet_files(pe)
```

Comment:
Hmm I would suggest renaming to `send_via_ftps()` to be consistent and to avoid confusion with the callback in the Orthanc plugin.

Reply:
Yeah agreed, as its not an orthanc plugin we should be snake case.

I'd also be tempted to just have a single `export-patient-data` API endpoint which writes the radiology report and exports it via FTPS (calling a`send_via_ftps` function so that we can keep the same tests as we've got currently)